src/GovPollingService.js

```diff
@@ -0,0 +1,31 @@
+import { PrivateService } from '@makerdao/services-core';
+import { POLLING } from './utils/constants';
+
+export default class GovPollingService extends PrivateService {
+  constructor(name = 'govPolling') {
+    super(name, ['smartContract']);
+  }
+
+  async createPoll(startDate, endDate, multiHash, url) {
+    const txo = await this._pollingContract().createPoll(
+      startDate,
+      endDate,
+      multiHash,
+      url
+    );
+    const pollId = parseInt(txo.receipt.logs[0].topics[2]);
+    return pollId;
```

Comment:
Nice, so in the UI we can display this and the admin user can copy it and enter it into the CMS